lib/tapioca/loaders/dsl.rb

```diff
@@ -70,17 +69,6 @@ def load_application
 
         say("Done", :green)
       end
-
-      sig { void }
-      def abort_if_pending_migrations!
-        return unless File.exist?("#{@app_root}/config/application.rb")
-        return unless defined?(::Rake)
```

Comment:
Removed this line as well as per https://github.com/Shopify/tapioca/pull/1418#discussion_r1119142551